client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -864,9 +864,12 @@ export const closeReprintDialog = ( orderId, siteId ) => {
 
 export const confirmReprint = ( orderId, siteId ) => ( dispatch, getState ) => {
 	dispatch( { type: WOOCOMMERCE_SERVICES_SHIPPING_LABEL_CONFIRM_REPRINT, orderId, siteId } );
-	const shippingLabel = getShippingLabel( getState(), orderId, siteId );
+	const state = getShippingLabel( getState(), orderId, siteId );
+
+	const label = find( state.labels, { label_id: state.reprintDialog.labelId } );
+	const isReturn = label.returning_label_id != null;
```

Comment:
Huh, I thought `eslint` is supposed to complain about not using `!==` here. Using `Boolean( label.returning_label_id )` or just `label.returning_label_id` (since it's gonna be used in a conditional anyway) would be more what we're used to if it can be either `null` or `undefined`.